package.json

```diff
@@ -153,7 +153,7 @@
     "typed-emitter": "^2.1.0",
     "typescript": "^5.6.2",
     "vhtml": "^2.2.0",
-    "which": "^4.0.0",
+    "which": "4.0.0",
```

Comment:
`which` has new major update `v5`, but we did not upgrade because an `engine` field is not compatible with the current Marp CLI.